DEVX-2281: Beginner Cloud CLI tutorial improvements: fix documentatio…

ybyzek opened the pull request:
…n issues

### Description 

https://confluentinc.atlassian.net/browse/DEVX-2281
https://github.com/confluentinc/examples/issues/869

_What behavior does this PR change, and why?_

Fixes bugs in the tutorial

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

<!-- Uncomment any of the following that are required -->
- [x] Documentation
<!-- - [ ] ccloud -->
- [x] ccloud/beginner-cloud
<!-- - [ ] ccloud/ccloud-stack -->
<!-- - [ ] clickstream -->
<!-- - [ ] clients/avro -->
<!-- - [ ] clients/cloud -->
<!-- - [ ] cloud-etl -->
<!-- - [ ] connect-streams-pipeline -->
<!-- - [ ] cp-quickstart -->
<!-- - [ ] kubernetes/gke-base -->
<!-- - [ ] kubernetes/replicator-gke-cc -->
<!-- - [ ] microservices-orders -->
<!-- - [ ] multi-datacenter -->
<!-- - [ ] multiregion -->
<!-- - [ ] music -->
<!-- - [ ] replicator-schema-translation -->
<!-- - [ ] replicator-security -->
<!-- - [ ] security/rbac -->
<!-- - [ ] security/secret-protection -->


### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

I have validated a local docs build and compared to tutorial run, so looking just for visual inspection

<!-- Uncomment any of the following that are required -->
<!-- - [ ] Documentation -->
<!-- - [ ] ccloud -->
<!-- - [ ] ccloud/beginner-cloud -->
<!-- - [ ] ccloud/ccloud-stack -->
<!-- - [ ] clickstream -->
<!-- - [ ] clients/avro -->
<!-- - [ ] clients/cloud -->
<!-- - [ ] cloud-etl -->
<!-- - [ ] connect-streams-pipeline -->
<!-- - [ ] cp-quickstart -->
<!-- - [ ] kubernetes/gke-base -->
<!-- - [ ] kubernetes/replicator-gke-cc -->
<!-- - [ ] microservices-orders -->
<!-- - [ ] multi-datacenter -->
<!-- - [ ] multiregion -->
<!-- - [ ] music -->
<!-- - [ ] replicator-schema-translation -->
<!-- - [ ] replicator-security -->
<!-- - [ ] security/rbac -->
<!-- - [ ] security/secret-protection -->

ybyzek committed:
DEVX-2281: Beginner Cloud CLI tutorial improvements: fix documentation issues

awalther28 commented on ccloud/docs/beginner-cloud.rst:
```diff
@@ -233,29 +231,23 @@ Produce and consume records with Confluent Cloud CLI
+      ccloud kafka topic produce demo-topic-1
 
-#. Verify your output resembles:
+#. The CLI waits for you to type data at the prompt, so type a few characters each on a new line. For example, type th enumbers 1 through 5
```
```suggestion
#. The CLI waits for you to type data at the prompt, so type a few characters each on a new line. For example, type the numbers 1 through 5:
```

awalther28 replied:
Just to note, the start.sh script produces numbers 1-10.

ybyzek replied:
@awalther28 good catch.  Dropping script down to 5 as well.

awalther28 commented on ccloud/docs/beginner-cloud.rst:
```diff
@@ -264,23 +256,18 @@ Produce and consume records with Confluent Cloud CLI
 
       Starting Kafka Consumer. ^C or ^D to exit
-      2
+      1
```
Just to note, the start.sh script consumes numbers 1-10.

ybyzek replied:
@awalther28 good catch.  Dropping script down to 5 as well.

awalther28 commented on ccloud/docs/beginner-cloud.rst:
```diff
@@ -328,14 +315,15 @@ Create a new service account with an API key/secret pair
        security.protocol=SASL_SSL
        bootstrap.servers=pkc-4kgmg.us-west-2.aws.confluent.cloud:9092
-       sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="ESN5FSNDHOFFSUEV" password\="nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB";
+       sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username='ESN5FSNDHOFFSUEV' password='nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB';
```
This also differs in the start.sh

ybyzek replied:
👍 and resolved

awalther28 reviewed (commented):
Thanks for the updates. I built the docs locally and used the start.sh to validate the output.
I think with a few of these changes should also make it to the start.sh, namely changing the number of ints produced and consumed AND the 'sasl.jaas.config' change (also mentioned in comments in review).

ybyzek committed:
Drop number messages in script down to 5

ybyzek committed:
Update ccloud/docs/beginner-cloud.rst

Co-authored-by: Allison Walther <12937975+awalther28@users.noreply.github.com>

ybyzek committed:
Replace quotes for sasl.jaas.config in script

ybyzek commented:
> Thanks for the updates. I built the docs locally and used the start.sh to validate the output.
> I think with a few of these changes should also make it to the start.sh, namely changing the number of ints produced and consumed AND the 'sasl.jaas.config' change (also mentioned in comments in review).

Thank you for the review @awalther28 .  I believe I have addressed all feedback.

awalther28 reviewed (approved):
@ybyzek looks like you have :) I don't have anything else to add
